### ISSUE TITLE
[S3 Discussion] Social Traps

### ISSUE THREAD

**Issue by tarngerine** [sunday-school/cybernetics-club#7]
John Platt, applies cybernetic concepts (e.g. feedback) to wicked problems

http://web.mit.edu/curhan/www/docs/Articles/15341_Readings/Negotiation_and_Conflict_Management/Platt_1973_Social_traps.pdf

**Comment by frnsys** [comment]
- **social trap** - when individual decisions aggregate to cause a situatoin that none of the participating individuals want and that is difficult to get out of, e.g. tragedy of the commons
  - climate change: no one wants to make the earth uninhabitable but people are still doing it
- **social fence** - the converse to a social trap, a "countertrap", where individuals decide _not_ to do something that would benefit everyone in the aggregate, e.g. bystander effect

---

Platt formalizes social traps and fences in terms of Skinner reinforcement:

**Skinner reinforcement:** situation or stimulus S, where a subject emits some behavior B, followed by some reinforcement or result R. R+ is a positive reinforcement (reward), makes the initial behavior B more probable when S occurs. R- is negative reinforcement (punishment), making B less probably when S occurs.

Platt makes a distinction between short-term R (RS) and long-term R (RL)

In these terms, a social trap can be described as: S => B => RS+ => RL-

That is, a stimulus elicits a behavior and there is a short-term reward (encouraging that behavior) but ultimately causes a long-term punishment

A fence can be described as:

S -> B -> RS- -> RL+

That is, a stimulus elicits a behavior and there is a short term punishment (discouraging that behavior), but there would be a long-term reward

"Reversal of reinforcers" can also be applied to traps, where it isn't a comparison of short/long-term, but rather that personal reward/punishment (RP+ or RP-) equates to group punishment/reward (RG- or RG+). e.g. something that rewards the individual is ultimately to the disadvantage of the group.

---

Three major types of traps/countertraps:
1. One person traps ("self-traps")
2. Missing-hero type (group trap) - where one person is needed to act for the group
3. Commons type (group trap) - where the common pursuit of individual goods leads to collective bads
- one person traps
  - simple delay reversals (RS+ => RL-)
    - examples: smoking, overeating
  - the countertrap of this type are (RS- => RL+)
    - example: saving for old age - eventually leads to reward, but deprives present pleasures
  - ignorance of the unexpected or reversed outcome
    - example: a fish swimming into a fish trap without realizing that he won't be able to get out
  - sliding reinforcers: reinforcers that change steadily as you go on repeating a behavior, so that it becomes less and less rewarding and eventually punishing, however you continue with the habit hoping that things will become rewarding again
    - example: drug addiction, using a lot of natural resources
- missing hero traps
  - when group profit (RG+) is blocked by RP- for any personal action
    - example: "the mattress problem" - a mattress falls into the middle of the road, everyone tries to drive around it, no one bothers getting out and moving it.
- commons traps
  - where RG- follows because of an excessive number of RP+ practitioners
  - this type of group trap cannot be solved by one or two heroes
  - example: prisoner's dilemma, Shubik's dollar game (a group of people bid on the dollar, the winner gets the dollar, but the winner and the second-highest bidder must both pay their bids)

---

Locked-in aspects of collective behavior: immediate small reinforcements or punishments lead to self-maintaining behavior

Three types of locked-in patterns in collective behavior:
1. Invisible hand - stabilization/equilibrium in the absence of any overt/mechanical causal mechanism (e.g. Adam Smith's invisible hand of the marketplace)
2. Invisible fist - competing between individuals does not lead to stabilization but instead deviates to an extreme past some point of no return (e.g. arms races)
3. Invisible chain - "a loop of transactional relationships among two or more people, forming self-maintaining systems that are sometimes very damaging and very hard to get out of" (pp647). i.e. a system where each member's behavior reinforces each others' behaviors

---

ways out of social traps:
- convert the delay into a shorter one (i.e. translate the long-term effects into short-term ones)
  - example: cigarette health warnings on cigarette packages
- add counterreinforcers, such as social incentives or punishments (i.e. adding new RS+ or RS-)
  - example: punitive laws
- change the nature of the long-run consequence (RL-)
  - that is, if possible, make the long-run negative consequence a positive one or remove it (probably through technological means...such as better medicine)
- add RS- for competing behavior (basically, substitution or systems of personal reward)
  - example: drink diet cola instead of sugar cola, rewarding yourself for studying
- get outside help in changing the reinforcement patterns of locked-in loops
  - that is, if locked in a loop, get someone else to come in and help you break it.
  - example: therapists, diet counselors
- set up superordinate authority to present entrapments, to allocate resources, to mediate conflicts, and to redirect immediate reinforcement patterns to more rewarding long-range goals
  - basically, set up governing or managing bodies to keep things on track
  - example: the fish and game commissions to prevent the exhaustion of game

---

**Nested traps**: essentially, traps that are intertwined with each other

**Comment by michaelpace** [comment]
```
Social Traps, John Platt, applies cybernetic concepts (e.g. feedback) to wicked problems

---

social traps
    definition: "The term refers to situations in society that contain traps formally
        like a fish trap, where men or organizations or whole societies
        get themselves started in some direction or some set of relationships
        that later prove to be unpleasant or lethal and that they see no
        easy way to back out of or to avoid."
    example: world population problem, where each family may find pleasure and
        advantage in more babies; and the problem of competitive consumption
        of nonrenewable natural resources.

countertrap
    converse of social trap
    definition: self-interest prevents individuals from acting in a way that would
        be of value to the whole group.
    example: kitty genovese murder in nyc in which 30+ neighbors watched a rape &
        murder out their windows without calling the police, because then
        they'd probably have to testify in court and there's a chance of
        the murderer or his friends hunting you down. everyone assumes someoone
        else'd call the police.

formalizing skinner
    S-B-R sequence
        S is stimulus
        B is behavior as result of stimulus
        R is reinforcement
        experimenter introduces S and R; B is supplied by the subject
    positive reinforcement (R+) makes B more likely to happen in response to
        B in the future
    negative reinforcement (R-) makes B less likely to happen in response to
        B in the future
    often written on two lines, with the experimenter on the bottom and the 
        subject on the top:
          B       B
        S   R   S   R

applying skinner formalizations to social traps
    social traps depend on the difference between the personal / short-term
        reinforcements for a given B and teh group consequences or long-term
        consequences of that B.
    "A social trap occurs, then, when there is an opposition between the highly
        motivating short-run reward or punishment, Rs+ or Rs-, and the long-
        run consequences, Rl+ or Rl-"
    then, a trap can be defined as:
          B             
        S   Rs+ ...  Rl-
    and a countertrap as:
          B             
        S   Rs- ...  Rl+
    three major types of traps tehy found:
        one-person traps / self-traps
        group traps / missing-hero traps / kitty genovese
        commons traps

one-person traps
    "The most important subgroup of one-person traps seems to involve the
        simple reversal of rein- forcers after a time delay."
    examples:
        smoking cigarettes, overeating
    examples of countertraps:
        saving for old age
    "A second subgroup of the one-person traps is that in which the problem
        is not simple delay, but rather ignorance of the unexpected or
        reversed outcome. The fish swimming into the fish trap does not
        know that he cannot get out. In the long run, ignorance is as
        lethal as evil. This is the case of the man handling a gun who
        shoots himself or his friend because he "didn't know it was
        loaded.""
    "Another subgroup is that of sliding reinf&rcers. These are reinforcers
        that change steadily as you go on repeating a behavior, so that
        it becomes less and less rewarding and in fact punishing"
        an example: drug addiction.
        another: "Once, large families with more babies were good for
            survival, and they were a delight, but now excessive
            babies have turned into an expense and have contributed
            to overcrowding for everyone"

missing hero traps
    "When group profit, Rg+, is blocked by Rp- for any personal action, we have
        the missing-hero trap."
    example: mattress falls on the highway, causing cars to slow down and
        go around it. no one wants to move the mattress since it'll be
        personally dangerous, and by the time you get to it, you can get
        past it so you have no incentive to move it any longer.
    another: reluctance of anyone to testify against the mafia.

commons traps
    Rg- only fallows becuase of hte excessive number of Rp+ practicioners.
    similar to everyone's cows grazing a commons, and then the grass running out
        for everyone.
    "The problemc annot be solved by one or two heroes volunteering not to
        graze their cows on the commons, although such a course is frequently
        advocated by men of good-will. And the problem is not the result of
        any single person doing anything that is unethical or bad, for if the
        number of persons involved were kept small, one can imagine that the
        collective good would be well served by the sum of all personal Rp+
        rewards. It is when the number is excessive that the difficulty
        arises."
    example: prisoner's dilemma. "This is one of the types of two-person
        non-zero-sum games ... Two prisoners have been caught by the police
        in some misdemeanor but who are suspected of worse crimes. They are
        held incommunicado from each other and each is questioned. The police
        offer a pattern of rewards such that if they both "talk" or "defect"
        on each other, they get the standard sentence for their crime; if they
        "cooperate" with each other, so that neither talks, they get off lightly
        for their misdemeanor; but if one talks and the other does not, the
        first gets a reward and goes free, while the second gets a doubly
        severe sentence. In this situation, the payoff matrix is designed by
        the police so that each man benefits by defecting, no matter what his
        partner does."
    "Generally, in prisoner's dilemma situations, it is found that the opposing
        players tend to lock into either steady cooperation or steady
        conflict with each other. Which pattern is obtained seems to
        depend critically on the outcome—or should we say the "reinforcement"
        of the first few plays. Sometimes a pattern of cooperation
        is quickly experienced as mutually profitable and is kept.
        But if such a pattern is not started early, it seems to be almost
        impossible for anyone to continue to cooperate when his opponent
        is continually defecting on him and making money at his expense.
        It is hard to keep working for Rg+ when the other party's behavior
        keeps turning it into Rp- for you."
    "As Rapoport (1971) has emphasized, this dilemma and these alternative
        outcomes are remarkably parallel to some aspects of international
        relations in the non-zero-sum situations of either mutual economic
        dependence or mutual nuclear threat. The United States and Canada
        have had locked-in cooperation; the United States and Russia
        have had 25 years of locked-in hostility and arms escalation."
    another example: sell-a-dollar. dinner party game with four rules. first,
        bidding starts at a nickel. second, bidding must go up by 5 cents
        per bid. third, bidding must not go over $50. fourth, since everyone
        will try to get the dollar for so little, the auctioneer gets the two
        highest bids, although only the highest bid gets the dollar.
    sell-a-dollar has a narrative. the first of two pivotal moments is when bidding
        passes 50 cents. then the auctioneer will be getting back more than
        the dollar.
        the second moment is when bidding passes $1. this is an important
        "locked-in" moment. the other person ahs bid a dollar, and you've bid
        95 cents. if you raise to $1.05, even if you win, you will lose money.
        but if you don't raise, you will lose a lot more. so you raise.

locked-in aspects of collective behavior
    three types of locked-in patterns
    1. adam smith's "invisible hand" of marketplace
        "He used this term to emphasize the absence of any overt or mechanical
        causal mechanism in the stabilization of prices or wages around
        some median value in a free economic market of competing individual.
        A similar invisible hand tends to equalize and centralize the
        political parties in systems with majority (rather than proportional
        representation) elections."
    2. "invisible fist"
        "competition of numerous individuals does not produce agreement
        on a median value, but instead runs away from the median, with
        either escalation or elimination past some point of no return."
    3. "invisible chain" (e.g., bane of colin's existence)
        "a loop of transactional relationships among two or more people,
        forming self-maintaining systems that are sometimes very damaging
        and very hard to get out of."

ways to prevent or get out of social traps
    1. change the delay to convert long-range consequences into more immediate
        ones
        example: putting labels on cigarette packages
    2. add counterreinforcers, such as social incentives or punishments, to
        encourage or discourage behaviors by their immediate Rs+ or Rs-.
        example: punitive laws
    3. change the nature of the long-run consequence, Rl-.
        this happens sometimes e.g. through new inventions or technology.
    4. add Rs+ for competing behavior, which will not lead to the bad long-range
        consequences.
        example: drink a diet cola with saccharine instead of fattening sugar.
    5. get outside help in changing the reinforcement patterns of locked-in loops.
        example: "The delinquent child gets reinforced for his behavior
        by the attention he gets in being scolded, the excitement of
        being chased by the police, or the admiration of his friends;
        and the teacher, parents, police, friends, and the child are
        caught together in an invisible chain of self-maintaining reinforcement
        transactions. Tharp and Wetzel tried to find "mediators" — a teacher
        or adult friend who could see the child's daily behavior and
        give him immediate reinforcers, such as marks in a book for
        increased attention or reading, with so many marks entitling
        him to extra television viewing or horseback riding on weekends.
        As the child's behavior began to change within a few weeks,
        teacher, police, and parents changed their attitudes, and his
        friends began to admire him for different things (and were reinforced
        by the network for their change of values also). At this point,
        Tharp and Wetzel, as the outside "therapists," were able to
        withdraw because the system had been flipped to a new
        self-maitaining mode."
    6. set up superordinate authority to present entrapments, to allocate resources,
        to mediate conflicts, and to redirect intermediate reinforcement
        patterns to more rewarding long-range goals.
        example: "a sheriff system with mayors and courts in a western
        frontier town all represent something more than just an outside
        therapist. They represent the demo- cratic creation of new superordinate
        authority able to manage and correct social traps that were leaing
        to collective bads."

nested traps
    mixed traps and nested traps are hard to solve
    example: invisible fist of media promoting violence, locking in to a
        multiplication of violent acts and violent individuals in the community.

```
